### DISCUSSION TITLE
NULL values when exporting  bytea fields - PostgreSQL

### DISCUSSION THREAD

**Discussion by merouanekhalili** [dbeaver/dbeaver#12173] [Q&A]
#### System information: 
- Operating system (distribution) and version : Ubuntu 20.10
- DBeaver version : 21.0.1.202103211229 CE
- Additional extensions

#### Connection specification:
- Database name and version : PostgreSQL 9.6.20
- Driver name : postgresql-42.2.5.jar
- Do you use tunnels or proxies (SSH, SOCKS, etc)?  No

#### Describe the problem you're observing:
When I try to export a table with **bytea** fields as an SQL insert, I got NULL as value for all **bytea** fields

#### Steps to reproduce, if exist:

CREATE TABLE public.mytable (
	myfield1 bytea NOT NULL,
	myfield2 bytea NOT NULL,
	created_at timestamptz NOT NULL,
	updated_at timestamptz NOT NULL,
	CONSTRAINT mytable_pk PRIMARY KEY (myfield1)
);

INSERT INTO public.mytable
(myfield1, myfield2, created_at, updated_at)
VALUES(decode('3036363263397A0C','hex'), decode('00000000000000000000000000000000','hex'), now(), now());

Right click on the table -> Tools -> Export Data -> SQL

**Comment by LonwoLonwo** [comment]
Hello @merouanekhalili 
Have you try to change data transfer Binaries settings?

![2021-04-20 12_40_30-Data Transfer](https://user-images.githubusercontent.com/45152336/115374686-bfa06d00-a1d5-11eb-9137-a357af8673be.png)

**Comment by merouanekhalili** [comment]
Hello @LonwoLonwo 

Thank you for your comment.

It is not a bug, I just forgot to change the settings for the data transfer binaries.
